api/src/main/java/org/openmrs/Encounter.java

```diff
@@ -384,6 +386,22 @@ public Patient getPatient() {
 	public void setPatient(Patient patient) {
 		this.patient = patient;
 	}
+
+	/**
+	 * Gets the set of diagnoses
```

Comment:
Just like you have done before, leave a line between the method description and its params documentation